src/main/java/anichan/parser/BookmarkParser.java

```diff
@@ -219,12 +232,25 @@ private void setSingleParameter(String paramGiven) throws AniException {
      * @throws AniException if there is more than 2 parameter provided
      */
     private String[] getSplitDescription(String description) throws AniException {
-        String[] paramGiven = description.split(DASH_PARAM);
+        String[] paramGiven = description.split(DASH);
         if (paramGiven.length > 2) {
             String invalidDescription = DESCRIPTION_ERROR_HEADER + description + TOO_MUCH_FIELDS;
             LOGGER.log(Level.WARNING, BOOKMARK_LOAD_ERROR_HEADER + invalidDescription);
             throw new AniException(invalidDescription);
         }
         return paramGiven;
     }
+
+    /**
+     * Checks and prevents parameter stacking e.g. "1-e 5" or "1-n test".
+     *
+     * @param firstParam first parameter before -
+     * @throws AniException if there is no space between the parameter
+     */
+    private void checkForParamStacking(String firstParam) throws AniException {
+        //Forgive the last param
+        if (!firstParam.endsWith(" ")) {
```

Comment:
Looks good, but I noticed some characters were refactored as constants while this `SPACE `weren't.

Reply:
Oh my bad, I missed it.